src/app/phast/phast-report/executive-summary/executive-summary.component.html

```diff
@@ -13,16 +13,33 @@ <h4 class="show-print print-section-header">Assessment Executive Summary</h4>
         <td class="percent-savings">Percent Savings (%)</td>
         <td [ngClass]="{'selected-modification': selectedModificationIndex == -1}">&mdash; &mdash;</td>
         <td *ngFor="let mod of modifications; let index = index;" [ngClass]="{'selected-modification': index == selectedModificationIndex}">
-
-          <div class="hide-print" *ngIf="mod.percentSavings">
-            
+          <div class="alert-danger" *ngIf="!phast.modifications[index].phast.valid.isValid">
```

Comment:
In here you should be able to use the "mod" object from the ngFor... i **think**

Reply:
That mod is a summary of results. If I keep as is, I won't have to modify the results array by adding 'valid' and assigning. Not sure which is preferable though.